jetty-server/src/main/config/etc/jetty-gzip.xml

```diff
@@ -18,8 +18,8 @@
         <Set name="dispatcherTypes" property="jetty.gzip.dispatcherTypes"/>
         <Set name="includedMethodList" property="jetty.gzip.includedMethodList"/>
         <Set name="excludedMethodList" property="jetty.gzip.excludedMethodList"/>
-        <Set name="includedMimeTypes" property="jetty.gzip.includedMimeTypeList"/>
-        <Set name="excludedMimeTypes" property="jetty.gzip.excludedMimeTypeList"/>
+        <Set name="includedMimeTypesList" property="jetty.gzip.includedMimeTypeList"/>
+        <Set name="excludedMimeTypesList" property="jetty.gzip.excludedMimeTypeList"/>
```

Comment:
I would have preferred to split the string in the XML, but we can't if we maintain the property attribute ability to not set if not set.